src/services/WolfDen.Application/Requests/Queries/Attendence/AttendanceSummary/AttendanceSummaryQueryHandler.cs

```diff
@@ -0,0 +1,131 @@
+﻿using MediatR;
+using Microsoft.EntityFrameworkCore;
+using WolfDen.Application.DTOs.Attendence;
+using WolfDen.Domain.Entity;
+using WolfDen.Domain.Enums;
+using WolfDen.Infrastructure.Data;
+
+namespace WolfDen.Application.Requests.Queries.Attendence.AttendanceSummary
+{
+    public class AttendanceSummaryQueryHandler : IRequestHandler<AttendanceSummaryQuery, AttendanceSummaryDTO>
+    {
+
+
+        private readonly WolfDenContext _context;
+
+        public AttendanceSummaryQueryHandler(WolfDenContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<AttendanceSummaryDTO> Handle(AttendanceSummaryQuery request, CancellationToken cancellationToken)
+        {
+            int minWorkDuration = 360;
+
+            DateOnly monthStart = new DateOnly(request.Year, request.Month, 1);
+            DateOnly monthEnd = monthStart.AddMonths(1).AddDays(-1);
+
+            AttendanceSummaryDTO summaryDto = new AttendanceSummaryDTO
+            {
+                Present = 0,
+                Absent = 0,
+                IncompleteShift = 0,
+                RestrictedHoliday = 0,
+                NormalHoliday = 0,
+                WFH = 0,
+                Leave = 0
+            };
+
+            DateOnly today = DateOnly.FromDateTime(DateTime.Now);
```

Comment:
DateTime.Now will use the timezone of the running server where the code is going to be executed. this could vary, use DateTime.UtcNow